gwpy/utils/tests/test_lal.py

```diff
@@ -52,11 +53,36 @@ def test_find_typed_function():
     try:
         import lalframe
     except ImportError:  # no lalframe
-        pass
-    else:
-        utils_lal.find_typed_function(
-            'REAL4', 'FrStreamRead', 'TimeSeries',
-            module=lalframe) is lalframe.FrStreamReadREAL4TimeSeries
+        return
+    utils_lal.find_typed_function(
+        'REAL4', 'FrStreamRead', 'TimeSeries',
+        module=lalframe) is lalframe.FrStreamReadREAL4TimeSeries
+
+
+@pytest.mark.parametrize(
+    'dtype',
+    [
+        numpy.int16, numpy.int32, numpy.int64,
+        numpy.uint16, numpy.uint32, numpy.uint64,
+        numpy.float32, numpy.float64,
+        numpy.complex64, numpy.complex128,
```

Comment:
Please move these to one per line, so that other types can be added inline with minimal diffs.

```suggestion
        numpy.int16,
        numpy.int32,
        numpy.int64,
        numpy.uint16,
        numpy.uint32,
        numpy.uint64,
        numpy.float32,
        numpy.float64,
        numpy.complex64,
        numpy.complex128,
```